scripts/installation/install-deps-alpine.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/sh
+
+# Comprehensive dependency installation script for Tracee (Alpine Linux)
+# Replaces .github/actions/build-dependencies for containerized workflows
+
+set -e
+
+# Source lib.sh for consistent logging and utilities  
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+# shellcheck disable=SC1091
+. "$SCRIPT_DIR/../lib.sh"
+
+info "Starting Tracee dependency installation on Alpine Linux"
+
+# Configuration
+GOLANG_VERSION="1.24.0"
+STATICCHECK_VERSION="2025.1"
+REVIVE_VERSION="v1.7.0"
+GOIMPORTS_REVISER_VERSION="v3.8.2"
+ERRCHECK_VERSION="v1.9.0"
+
+install_base_packages() {
+    info "Installing base packages"
+    require_cmds apk
+    
+    apk update
+    apk add --no-cache \
+        bash \
+        build-base \
+        sudo \
+        coreutils \
+        findutils \
+        git \
+        curl \
+        rsync \
+        make \
+        gcc \
+        musl-dev \
+        linux-headers \
+        elfutils-dev \
+        libelf-static \
+        zlib-static \
+        zstd-static \
+        libc6-compat \
+        tar \
+        ca-certificates
+
+    # Create symlinks for compatibility with tests expecting binaries in /usr/bin
+    # BusyBox applets - link directly to busybox so applet name is detected correctly
+    ln -sf /bin/busybox /usr/bin/uname
+    ln -sf /bin/busybox /usr/bin/date
+
+    info "Base packages installed successfully"
+}
+
+install_golang() {
+    info "Installing Go ${GOLANG_VERSION}"
+    require_cmds curl tar
+    
+    # Detect architecture for Go download
+    ARCH=$(uname -m)
+    case "$ARCH" in
+        x86_64) GOARCH="amd64" ;;
+        aarch64) GOARCH="arm64" ;;
+        *)
+            error "Unsupported architecture: $ARCH"
+            exit 1
+            ;;
+    esac
+
+    # Remove any existing Go installation
+    rm -f /usr/bin/go /usr/bin/gofmt
+    rm -rf /usr/local/go
+    
+    # Download and install Go
+    curl -L -o /tmp/golang.tar.gz "https://go.dev/dl/go${GOLANG_VERSION}.linux-${GOARCH}.tar.gz"
+    tar -C /usr/local -xzf /tmp/golang.tar.gz
+    rm /tmp/golang.tar.gz
+    
+    # Create symlinks
+    ln -s /usr/local/go/bin/go /usr/bin/go
+    ln -s /usr/local/go/bin/gofmt /usr/bin/gofmt
+    
+    # Verify installation
+    go version
+    info "Go ${GOLANG_VERSION} installed successfully"
+}
+
+install_clang() {
+    info "Installing Clang using centralized script"
+    require_cmds bash
+    
+    # Call our existing Clang installation script
+    bash "$SCRIPT_DIR/install-clang.sh"
+    
+    info "Clang installation completed"
+}
+
+install_go_tools() {
```

Comment:
future: extract to `install-go.sh`.

`./install-go.sh go`
`./install-go.sh tools`
`./install-go.sh go tools`